evalml/tests/automl_tests/test_automl.py

```diff
@@ -2956,396 +2571,36 @@ def test_data_splitter_multi(AutoMLTestEnv, X_y_multi):
         automl.search()
 
 
-@patch("evalml.tuners.skopt_tuner.SKOptTuner.add")
-def test_iterative_algorithm_pipeline_hyperparameters_make_pipeline_other_errors(
-    mock_add, AutoMLTestEnv, X_y_multi
-):
-    X, y = X_y_multi
-    custom_hyperparameters = {
-        "Imputer": {"numeric_impute_strategy": Categorical(["most_frequent", "mean"])}
-    }
-    estimators = get_estimators("multiclass", [ModelFamily.EXTRA_TREES])
-
-    component_graphs = {}
-    for ind, estimator in enumerate(estimators):
-        component_graphs[f"CG_{ind}"] = [estimator]
-
+def test_search_with_text(AutoMLTestEnv):
```

Comment:
This diff makes it look like a new addition but its the same old `test_search_with_text`!